src/test/java/racingcar/CarTest.java

```diff
@@ -0,0 +1,63 @@
+package racingcar;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class CarTest {
+	@Test
+	@DisplayName("자동차 이름은 5자 이하만 가능하다.")
+	void test() {
+		Assertions.assertThatIllegalArgumentException()
+			.isThrownBy(() -> new Car("junbo-car"));
+	}
+
+	@ParameterizedTest
+	@MethodSource("fourToNine")
+	@DisplayName("숫자가 4 이상이면 전진한다")
+	void fourToNineTest(Integer condition) {
+		Car car = new Car("kim");
+		car.move(new ConditionStrategy(condition));
+		assertThat(car.position()).isEqualTo(1);
+	}
+
+	@ParameterizedTest
+	@MethodSource("oneToThree")
+	@DisplayName("숫자가 4 미만이면 움직이지 않는다")
+	void oneToThreeTest(Integer condition) {
+		Car car = new Car("kim");
+		car.move(new ConditionStrategy(condition));
+		assertThat(car.position()).isEqualTo(0);
+	}
+
+	@Test
+	@DisplayName("움직이는 상황에서 움직인다")
+	void goTest() {
+		Car car = new Car("kim");
+		car.move(new GoStrategy());
```

Comment:
Test를 위한 Stub을 만들어 테스트하는 것도 괜찮은 방법이지만, 람다식을 활용해볼 수도 있을 것 같아요. 😉

```suggestion
		car.move(() -> true);
```